build_projects/dotnet-cli-build/TestTargets.cs

```diff
@@ -30,7 +30,7 @@ public class TestTargets : Task
             // "dotnet-compile-fsc.Tests",
             "dotnet-new.Tests",
             "dotnet-pack.Tests",
-            "dotnet-projectmodel-server.Tests",
+            //"dotnet-projectmodel-server.Tests",
```

Comment:
I can't wait until we:

1. remove the projectmodel-server from the product, and that allows
2. we delete these tests for good.